roxie/udplib/udptrr.cpp

```diff
@@ -806,20 +1441,28 @@ class CReceiveManager : implements IReceiveManager, public CInterface
     uid_map         collators;
     CriticalSection collatorsLock; // protects access to collators map
 
-  public:
+public:
     IMPLEMENT_IINTERFACE;
-    CReceiveManager(int server_flow_port, int d_port, int client_flow_port, int queue_size, int m_slot_pr_client, bool _encrypted)
-        : collatorThread(*this), encrypted(_encrypted), sendersTable([client_flow_port](const ServerIdentifier ip) { return new UdpSenderEntry(ip.getIpAddress(), client_flow_port);})
+    CReceiveManager(int server_flow_port, int d_port, int client_flow_port, int queue_size, bool _encrypted)
+        : collatorThread(*this), encrypted(_encrypted),
+        sendersTable([client_flow_port](const ServerIdentifier ip) { return new UdpSenderEntry(ip.getIpAddress(), client_flow_port);}),
+        input_queue_size(queue_size), receive_flow_port(server_flow_port), data_port(d_port)
     {
 #ifndef _WIN32
         setpriority(PRIO_PROCESS, 0, -15);
```

Comment:
I wondered about this and the other call, should these be part of the priorities check/set ?